web/src/containers/ActivityDetailAll.js

```diff
@@ -31,7 +32,7 @@ ActivityDetailAll.propTypes = {
 
 const mapStateToProps = ({ activities: { byId } }, { aId, num }) => {
   const activity = byId[aId];
-  const title = `Program Activities › ${activityTitle(activity, num)}`;
+  const title = `${t('activities.header')} › ${activityTitle(activity, num)}`;
```

Comment:
Not wild about this one...  I think something like:

```js
t('activities.header', { number: num, name: a.name })
```

would be preferable, but I wasn't sure if I18n supported conditional parameters.  That is, unclear if it could differentiate between the cases where the activity did and didn't have a name.  I suppose we could have `activities.header` and `activities.headerAnonymous`?

I dunno, but I'm not wild about splitting up the text this way, and it does leave some user-visible text in the code (the `>`).